platform/ios/src/MGLMapView.mm

```diff
@@ -1376,6 +1376,11 @@ - (void)touchesBegan:(__unused NSSet<UITouch *> *)touches withEvent:(__unused UI
     {
         [self setUserTrackingMode:MGLUserTrackingModeNone animated:NO];
     }
+    
+    if (!self.zoomEnabled && !self.pitchEnabled && !self.rotateEnabled && !self.scrollEnabled) {
+        return;
+    };
```

Comment:
Can you explain your decision to place this here (and not as the first thing in this method)?

Reply:
No specific reason 😅 I didn't realize it at the time, but it makes sense to move it up so that we can return earlier if need be.